ui/src/views/settings/SettingsRelay.vue

```diff
@@ -314,11 +411,98 @@ export default {
       );
     },
     setRelayConfigurationCompleted() {
+      // set always bcc
+      this.setAlwaysBcc();
+    },
+    async setAlwaysBcc() {
+      this.loading.setRelayConfiguration = true;
+      this.error.setRelayConfiguration = "";
+      this.error.bcc = "";
+      const taskAction = "set-always-bcc";
+      const eventId = this.getUuid();
+
+      // register to task error
+      this.core.$root.$once(
+        `${taskAction}-aborted-${eventId}`,
+        this.setAlwaysBccConfigurationAborted
+      );
+
+      // register to task validation
+      this.core.$root.$once(
+        `${taskAction}-validation-failed-${eventId}`,
+        this.setAlwaysBccConfigurationValidationFailed
+      );
+
+      // register to task completion
+      this.core.$root.$once(
+        `${taskAction}-completed-${eventId}`,
+        this.setAlwaysBccConfigurationCompleted
+      );
+
+      const res = await to(
+        this.createModuleTaskForApp(this.instanceName, {
+          action: taskAction,
+          data: {
+            bcc: this.isBccEnabled ? this.bcc : "",
+          },
+          extra: {
+            title: this.$t("action." + taskAction),
+            eventId,
+          },
+        })
+      );
+      const err = res[0];
+
+      if (err) {
+        console.error(`error creating task ${taskAction}`, err);
+        this.error.setRelayConfiguration = this.getErrorMessage(err);
+        this.loading.setRelayConfiguration = false;
+        return;
+      }
+    },
+    setAlwayBccConfigurationAborted(taskResult, taskContext) {
```

Comment:
Typo: let's rename this function to `setAlwaysBccConfigurationAborted`

Reply:
good catch